src/vs/server/node/remoteExtensionHostAgentServer.ts

```diff
@@ -768,7 +768,7 @@ export async function createServer(address: string | net.AddressInfo | null, arg
 		const hasVSDA = fs.existsSync(join(FileAccess.asFileUri('').fsPath, '../node_modules/vsda'));
 		if (hasVSDA) {
 			try {
-				return <typeof vsda>globalThis._VSCODE_NODE_MODULES['vsda'];
+				return require('vsda');
```

Comment:
Would we need `const require = createRequire(import.meta.url);` in this file?

Reply:
indeed